### ISSUE TITLE
Build fails on missing linux/if_xdp.h

### ISSUE THREAD

**Issue by netoptimizer** [xdp-project/xdp-tools#2]
On another machine, build fails due to missing ```linux/if_xdp.h```
```
$ make
[...]
make[3]: Entering directory '/home/jbrouer/git/xdp-tools/lib/util'
cc -I../libbpf/src -Wall -I../../headers -c -o params.o params.c
params.c:13:10: fatal error: linux/if_xdp.h: No such file or directory
 #include <linux/if_xdp.h>
          ^~~~~~~~~~~~~~~~
compilation terminated.
```

In the XDP-tutorial this issue is already solved, and we actually have two copies of this file:
```
[xdp-tutorial]$ find . | grep if_xdp
./libbpf/include/uapi/linux/if_xdp.h
./headers/linux/if_xdp.h
```

And in this XDP-tools repo it's indirectly available via libbpf submodule.

**Comment by netoptimizer** [comment]
As a small help, let me note, that on Fedora this header belongs to the package ```kernel-headers```.

```
$ rpm -qf /usr/include/linux/if_xdp.h
kernel-headers-5.2.18-100.fc29.x86_64
```

**Comment by tohojo** [comment]
Yeah, fixing the build system is next on my list :)

**Comment by netoptimizer** [comment]
This is fixed by commit 39da9bba
(Guess I used the wrong markup in commit message to get this picked up automagically)

**Comment by tohojo** [comment]
Jesper Dangaard Brouer <notifications@github.com> writes:

> This is fixed by commit 39da9bba
> (Guess I used the wrong markup in commit message to get this picked up
> automagically)

See
https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords
for the accepted syntax :)
